### PR TITLE
Build modelzooserver Docker image

### PR DESCRIPTION
**Pull request by Yancey0623**
This PR build and publish the model zoo server Docker image.
TODO: deploy model zoo server on playground.

### CODE REVIEW

File: scripts/travis/build.sh [hunk 1]
```diff
@@ -43,42 +43,22 @@ echo "Build sqlflow:ci byloading $TRAVIS_BUILD_DIR/build ..."
...
-    docker build --cache-from sqlflow/sqlflow:mysql -t sqlflow/sqlflow:mysql \
-           -f docker/mysql/Dockerfile "$TRAVIS_BUILD_DIR"
-else
-    docker build -t sqlflow/sqlflow:mysql \
-           -f docker/mysql/Dockerfile "$TRAVIS_BUILD_DIR"
-fi
+function build_sqlflow_image() {
+    echo "Build sqlflow:${1} by loading $TRAVIS_BUILD_DIR/build ..."
+    if docker pull sqlflow/sqlflow:"${1}" 2> /dev/null; then
+       echo " using sqlflow/sqlflow${1} as the cache image"
```
Comment (lhw362950217) [inline scripts/travis/build.sh]: sqlflow/sqlflow${1} -> sqlflow/sqlflow:${1}

File: docker/modelzooserver/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+FROM ubuntu:18.04
+
+# Choose faster mirrors for apt-get and pip
+COPY docker/dev/find_fastest_resources.sh /usr/local/bin/
+RUN /bin/bash -c 'source find_fastest_resources.sh \
+        && echo "Choose the fastest APT source ..." \
+        && choose_fastest_apt_source
+
+# Install pre-built SQLFlow components.
+COPY build /build
```
Comment (lhw362950217) [inline docker/modelzooserver/Dockerfile]: We do not need the whole directory?

File: docker/modelzooserver/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+FROM ubuntu:18.04
+
+# Choose faster mirrors for apt-get and pip
+COPY docker/dev/find_fastest_resources.sh /usr/local/bin/
+RUN /bin/bash -c 'source find_fastest_resources.sh \
+        && echo "Choose the fastest APT source ..." \
+        && choose_fastest_apt_source'
+
+# Install pre-built SQLFlow components.
+COPY build/modelzooserver /build/modelzooserver
```
Comment (lhw362950217) [inline docker/modelzooserver/Dockerfile]: can we just use `COPY build/modelzooserver /usr/local/bin/modelzooserver` ?

File: docker/modelzooserver/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+COPY docker/dev/find_fastest_resources.sh /usr/local/bin/
+RUN /bin/bash -c 'source find_fastest_resources.sh \
+        && echo "Choose the fastest APT source ..." \
+        && choose_fastest_apt_source'
+
+# Install pre-built SQLFlow components.
+COPY build/modelzooserver /usr/local/bin/modelzooserver
+
+# Install docker.io for building models
+RUN apt-get -qq update && apt-get -qq install -y axel sudo docker.io
```
Comment (typhoonzero) [inline docker/modelzooserver/Dockerfile]: Maybe put two `RUN` lines in one line to reduce the image layers.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**